testUtil/withNextRouter.js

```diff
@@ -6,9 +6,9 @@ import { RouterContext } from 'next/dist/next-server/lib/router-context'
 export function withTestRouter(tree, router = {}) {
   const {
     route = '',
-    pathname = '',
+    pathname = '/not-root',
```

Comment:
Why is this needed?

Reply:
It's not needed, and I should probably revert it back. Initially I intended to test redirections with `withTestRouter` and the base route should be different from `/` in case if you need check redirect to/from index file.